src/track/beatmap.cpp

```diff
@@ -22,7 +22,7 @@ inline double samplesToFrames(const double samples) {
 }
 
 inline double framesToSamples(const double frames) {
```

Comment:
Whoops, the proto defines frame_position as int32 so the argument here could be int and you wouldn't have to floor it.